acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationCachingIT.java

```diff
@@ -0,0 +1,519 @@
+                        "src/test/resources/multiple-site-based-policy-set.json", Effect.DENY },
+                { this.acsUrl, PETE_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/multiple-site-based-policy-set.json", Effect.DENY },
+                { this.acsUrl, JLO_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/multiple-site-based-policy-set.json", Effect.DENY },
+
+                { this.acsUrl, MARISSA_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.PERMIT },
+                { this.acsUrl, MARISSA_V1, null, "GET", new BaseResource("/secured-by-value/sites/ny"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.NOT_APPLICABLE },
+                { this.acsUrl, JOE_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.PERMIT },
+                { this.acsUrl, JOE_V1, null, "GET", new BaseResource("/secured-by-value/sites/ny"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.NOT_APPLICABLE },
+                { this.acsUrl, PETE_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.PERMIT },
+                { this.acsUrl, PETE_V1, null, "GET", new BaseResource("/secured-by-value/sites/ny"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.NOT_APPLICABLE },
+                { this.acsUrl, JLO_V1, null, "GET", new BaseResource("/secured-by-value/sites/sanramon"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.PERMIT },
+                { this.acsUrl, JLO_V1, null, "GET", new BaseResource("/secured-by-value/sites/ny"),
+                        "src/test/resources/single-site-based-policy-set.json", Effect.NOT_APPLICABLE },
+
+                { this.acsUrl, new BaseSubject(DEFAULT_SUBJECT_ID), null, NOT_MATCHING_ACTION,
+                        new BaseResource(DEFAULT_RESOURCE_IDENTIFIER),
+                        "src/test/resources/policies/permit-admin-only-using-condition.json", Effect.DENY },
+                { this.acsUrl, new BaseSubject(DEFAULT_SUBJECT_ID), null, NOT_MATCHING_ACTION,
+                        new BaseResource(DEFAULT_RESOURCE_IDENTIFIER), "src/test/resources/policies/deny-all.json",
+                        Effect.DENY } };
+        return data;
+    }
+
+    @DataProvider(name = "endpointProvider")
+    public Object[][] getAcsEndpoint() {
+        Object[][] data = new Object[][] { { this.acsUrl, this.acsZone1Headers }, { this.acsZone1Url, null } };
+        return data;
+    }
+}
```

Comment:
Data providers are not used for any of the tests.  These data providers could be used to refactor tests in AccessControlServiceIT